snuba/state/rate_limit.py

```diff
@@ -126,7 +126,7 @@ def rate_limit(
     pipe.zremrangebyscore(
         bucket, "-inf", "({:f}".format(now - rate_history_s)
     )  # cleanup
-    pipe.zadd(bucket, now + state.max_query_duration_s, query_id)  # add query
+    pipe.zadd(bucket, now + state.max_query_duration_s, query_id)  # type: ignore
```

Comment:
Why ignore? Is is failing because there's something that we might not have typed correctly here?

Reply:
It's actually the redis library itself. These functions return the pipeline but they are typed as returning Any. Rather than attempt to type check every chained call it seemed better to ignore until the redis library updates.